app/views/handlers/feedback.py

```diff
@@ -174,3 +199,45 @@ def is_limit_reached(session_data: SessionData) -> bool:
     @staticmethod
     def is_enabled(schema: QuestionnaireSchema) -> bool:
         return schema.get_submission().get("feedback")
+
+
+class FeedbackMetadata:
+    def __init__(self, feedback_count, form_type, language_code, region_code, tx_id):
+        self.feedback_count = feedback_count
+        self.form_type = form_type
+        self.language_code = language_code
+        self.region_code = region_code
+        self.tx_id = tx_id
+
+    def __call__(self) -> Mapping:
+        return {
+            "feedback_count": self.feedback_count,
+            "form_type": self.form_type,
+            "language_code": self.language_code,
+            "region_code": self.region_code,
+            "tx_id": self.tx_id,
+        }
+
+
+class FeedbackPayload:
+    def __init__(
+        self,
+        feedback_text,
+        feedback_type,
+        feedback_type_question_category=None,
+    ):
+        self.feedback_text = feedback_text
+        self.feedback_type = feedback_type
+        self.feedback_type_question_category = feedback_type_question_category
+
+    def __call__(self) -> Mapping:
+        payload = {
+            "feedback_text": self.feedback_text,
+            "feedback_type": self.feedback_type,
+        }
+        if self.feedback_type_question_category:
+            payload[
+                "feedback_type_question_category"
+            ] = self.feedback_type_question_category
+
+        return payload
```

Comment:
This can also be simplified to:
```suggestion
        payload = vars(self)
        if not self.feedback_type_question_category:
            del payload["feedback_type_question_category"]

        return payload
```